ner_telhub/widgets/menu_widgets.py

```diff
@@ -76,5 +78,39 @@ def __init__(self, parent: QWidget):
         layout = QVBoxLayout()
         layout.addWidget(table)
         self.setLayout(layout)
-        
+
+
+class ButtonStyles(Enum):
+    """Defines the styles of buttons. When hovering over them, they turn two shades brighter"""
+    # Lighter shades found with https://www.w3schools.com/colors/colors_picker.asp
+
+    DEFAULT = ""
+    RED = """QPushButton {color: white; background-color: #FF5656; border-radius: 4px; padding: 3% 8%;}
+        QPushButton:hover {background-color: #ff8080; color: white;}"""
+    GREEN = """QPushButton {color: white; background-color: #1a8f35; border-radius: 4px; padding: 3% 8%;}
+        QPushButton:hover {background-color: #20ac40; color: white;}"""
+    BLUE = """QPushButton {color: white; background-color: #0693E3; border-radius: 4px; padding: 3% 8%;}
+        QPushButton:hover {background-color: #1fa9f9; color: white;}"""
+    GRAY = """QPushButton {color: white; background-color: #999999; border-radius: 4px; padding: 3% 8%;}
+        QPushButton:hover {background-color: #b3b3b3; color: white;}"""
+
+
+
+class NERButton(QPushButton):
```

Comment:
Let's move this into its own file within the widget's directory, maybe called styled_widgets.py. I think we'll do this for more than just buttons so they all can be in this file